cmd/interactive.go

```diff
@@ -144,6 +149,11 @@ func generateInteractive(cmd *cobra.Command, opts runOptions) error {
 			scanner.Prompt.UseAlt = false
 			sb.Reset()
 
+			continue
+		case errors.Is(err, readline.ErrNewLineDetected):
+			sb.WriteString(line)
+			fmt.Fprintln(&sb)
```

Comment:
```suggestion
```

required?

Reply:
this adds the newline into the final prompt.